release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1265,6 +1265,34 @@ With this release, the CCO no longer checks for the nonexistent permission.
 [id="ocp-4-16-hosted-control-plane-bug-fixes_{context}"]
 ==== Hosted Control Plane
 
+* Previously, Multus CNI required CSRs to be approved when you used the `Other` network type in hosted clusters. The proper RBAC rules were set only when the the network type was `Other` and was set to Calico. As a consequence, the CSRs were not approved when the network type was `Other` and set to Cilium. With this update, the correct RBAC rules are set for all valid network types, and RBACs are now properly configured when you use the `Other` network type. (link:https://issues.redhat.com/browse/OCPBUGS-26977[*OCPBUGS-26977*])
+
+* Previously, an Amazon Web Services (AWS) policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-29391[*OCPBUGS-29391*])
+
+* Previously, in disconnected environments, the HyperShift Operator ignored registry overrides. As a consequence, changes to node pools were ignored, and node pools encountered errors. With this update, the metadata inspector works as expected during the HyperShift Operator reconciliation, and the override images are properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-34773[*OCPBUGS-34773*])
+
+* Previously, the Hypershift Operator was not using the `RegistryOverrides` mechanism to inspect the image from the internal registry. With this release, the metadata inspector works as expected during the Hypershift Operator reconciliation, and the `OverrideImages` are properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-32220[*OCPBUGS-32220*])
+
+* Previously, the OpenShift Cluster Manager container did not have the right TLS Certificates. As a result, image streams could not be used in disconnected deployments. With this update, the TLS Certificates are added as projected volumes. (link:https://issues.redhat.com/browse/OCPBUGS-34390[*OCPBUGS-34390*])
+
+* Previously, the `azure-kms-provider-active` container in the KAS pod used an entrypoint statement in shell form in the Dockerfile. As a consequence, the container failed. To resolve this issue, use the `exec` form for the entrypoint statement. (link:https://issues.redhat.com/browse/OCPBUGS-33940[*OCPBUGS-33940*])
+
+* Previously, the `konnectivity-agent` daemonset used the `ClusterIP` DNS policy. As a result, when CoreDNS was down, the `konnectivity-agent` pods on the data plane could not resolve the proxy-server-address, and they could fail to `konnectivity-server` in the control plane. With this update, the `konnectivity-agent` daemonset was modified to use `dnsPolicy: Default`. The `konnectivity-agent` uses the host system DNS service to look up the proxy server address, and it does not depend on CoreDNS anymore. (link:https://issues.redhat.com/browse/OCPBUGS-31444[*OCPBUGS-31444*])
```

Comment:
I was wondering about the markup for "proxy-server-address", but couldn't find any hits on that in the linked Jira / code fix PR (both mention "proxy-server-url") or QE test (mentions an actual `proxy-server-host` flag), so I wonder if this should more accurately/generically be "proxy server address" or "proxy server URL"? E.g.,:
```suggestion
* Previously, the `konnectivity-agent` daemonset used the `ClusterIP` DNS policy. As a result, when CoreDNS was down, the `konnectivity-agent` pods on the data plane could not resolve the proxy server URL, and they could fail to `konnectivity-server` in the control plane. With this update, the `konnectivity-agent` daemonset was modified to use `dnsPolicy: Default`. The `konnectivity-agent` uses the host system DNS service to look up the proxy server address, and it does not depend on CoreDNS anymore. (link:https://issues.redhat.com/browse/OCPBUGS-31444[*OCPBUGS-31444*])
```